docs/topics/multiplatform/multiplatform-android-layout.md

```diff
@@ -0,0 +1,108 @@
+[//]: # (title: Android source set layout)
+
+The new Android source set layout was introduced in Kotlin 1.8.0 and became default in 1.9.0. Follow our guide to
+understand the key differences between the deprecated and the new layout and migrate your projects.
```

Comment:
comma? 

>, and migrate your projects

Reply:
You got here: "Follow the guide ... between the deprecated **and** the new layout **and** migrate".
I'd recommend rephrasing the last part, WDYT?

Reply:
I don't know, it'a pretty standard phrasing with two 2 different pairs of homogeneous members